buildlib/pr/main.yml

```diff
@@ -392,7 +392,7 @@ stages:
           matrix:
             centos7:
               CONTAINER: centos7_ib
-              extra_modules: ucx-ib ucx-cma ucx-rdmacm ucx-ib-mlx5
+              extra_modules: ucx-ib ucx-cma ucx-rdmacm ucx-ib-mlx5 #ucx-ib-efa
```

Comment:
why commented?

Reply:
on some machines, the module is not built since efa library is not available, and anyways it is not needed. was adding for documentation purpose, but will remove it completely.